### PR TITLE
DOC: improve shared content between comparison pages

### PR DESCRIPTION
**Pull request by afeld**
This pull request does a few things between the SAS and Stata pages, in separate commits:

- Makes the headings match, where it makes sense for them to
- Create more shared includes, as a follow-up to https://github.com/pandas-dev/pandas/pull/38887
- Improves some wording and ensures more methods are linked in the comparison includes

The motivation here is that I'm working on adding the other sections to the Comparison to Spreadsheets page, and want to ensure they're consistent.

---

- [ ] ~~closes #xxxx~~
- [x] tests added / passed
- [ ] ~~passes `black pandas`~~
- [ ] ~~passes `git diff upstream/master -u -- "*.py" | flake8 --diff`~~
- [ ] ~~whatsnew entry~~

### CODE REVIEW

**Comment by jreback** [comment]
nice, ping on green.

**Comment by jreback** [comment]
thanks @afeld

File: doc/source/getting_started/comparison/includes/missing.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+This doesn't work in pandas.  Instead, the :func:`pd.isna` or :func:`pd.notna` functions
+should be used for comparisons.
+
+.. ipython:: python
+
+   outer_join[pd.isna(outer_join["value_x"])]
+   outer_join[pd.notna(outer_join["value_x"])]
+
+pandas also provides a variety of methods to work with missing data -- some of
+which would be challenging to express in Stata. For example, there are methods to
```
Comment (afeld) [inline doc/source/getting_started/comparison/includes/missing.rst:10]: Argh, missed the reference to Stata here. Fixed in https://github.com/pandas-dev/pandas/pull/38935.

File: doc/source/getting_started/comparison/comparison_with_stata.rst [hunk 2]
```diff
@@ -465,38 +428,13 @@ or the intersection of the two by using the values created in the
...
-
-   left_join = df1.merge(df2, on=["key"], how="left")
-   left_join
-
-   right_join = df1.merge(df2, on=["key"], how="right")
-   right_join
-
-   outer_join = df1.merge(df2, on=["key"], how="outer")
-   outer_join
+.. include:: includes/merge_setup.rst
```
Comment (afeld) [inline doc/source/getting_started/comparison/comparison_with_stata.rst:431]: Argh, another mistake. Also fixed in https://github.com/pandas-dev/pandas/pull/38935.

**Pull request merged**